pv_site_api/main.py

```diff
@@ -243,25 +277,81 @@ async def get_pv_actual(site_uuid: str, session: Session = Depends(get_session))
     if int(os.environ["FAKE"]):
         return await make_fake_pv_generation(site_uuid)
 
-    start_utc = get_start_datetime()
-    generations_sql = get_pv_generation_by_sites(
-        session=session, start_utc=start_utc, site_uuids=[uuid.UUID(site_uuid)]
+    return _get_generation_by_sites(session, site_uuids=[site_uuid])[0]
+
+
+@app.get("/sites/pv_actual", response_model=list[MultiplePVActual])
+def get_pv_actual_many_sites(
+    site_uuids: str,
+    session: Session = Depends(get_session),
+):
+    """
+    ### Get the actual power generation for a list of sites.
+    """
+    site_uuids_list = site_uuids.split(",")
+    return _get_generation_by_sites(session, site_uuids_list)
+
+
+def _get_latest_forecast_by_sites(session: Session, site_uuids: list[str]) -> list[Forecast]:
+    """Get the latest forecast for given site uuids.
+
+    Return:
+    ------
+        Pydantic models representing the data.
+    """
+    # Get the latest forecast for each site.
+    subquery = (
+        session.query(ForecastSQL)
```

Comment:
please use the read function in the pv-datamodel

Reply:
This is what it used to be
```
generations_sql = get_pv_generation_by_sites(
        session=session, start_utc=start_utc, site_uuids=[uuid.UUID(site_uuid)]
```

If you want to update it then please update in the datamodel

Reply:
```  
 latest_forecast_values = get_latest_forecast_values_by_site(
        session=session, site_uuids=[site_uuid], start_utc=start_utc
    )
```

was what used to be there